src/libsyntax/ast.rs

```diff
@@ -97,9 +97,12 @@ impl Path {
     }
 
     pub fn default_to_global(mut self) -> Path {
```

Comment:
Pre-existing, but I think a comment on this function would be great, just showing the transformations it does (and maybe explaining the reason for its existence).